packages/components/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,103 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type ForwardedRef, cloneElement, forwardRef, type MouseEvent, type ReactElement, type ReactNode } from 'react';
+
+import { buttonIconSizeMap, buttonStyles } from './styles/buttonStyles';
+import { iconStyles } from './styles/iconStyles';
+import { textStyles } from './styles/textStyles';
+
+type ButtonRounding = 'medium' | 'full';
+
+type ButtonStyle = 'default' | 'positive' | 'negative' | 'danger' | 'loud';
+
+type ButtonSize = 'small' | 'medium' | 'large';
+
+type ButtonType = 'button' | 'submit' | 'reset';
+
+type ButtonEmphasis = 'default' | 'weak' | 'strong';
+
+export type ButtonProps = {
+    type?: ButtonType;
+    title?: string;
+    style?: ButtonStyle;
+    emphasis?: ButtonEmphasis;
+    hideLabel?: boolean;
+    size?: ButtonSize;
+    rounding?: ButtonRounding;
+    disabled?: boolean;
+    icon?: ReactElement;
+    children?: ReactNode;
+    onClick?: (event?: MouseEvent<HTMLButtonElement>) => void;
+    hugWidth?: boolean;
+    'aria-label'?: string;
+    'aria-describedby'?: string;
+    formId?: string;
```

Comment:
We should enforce having the button _inside_ a form and not outside, what do you think about removing it?

Reply:
agree, that makes sense.

removed